cmd/containerd-shim/main_unix.go

```diff
@@ -66,6 +66,19 @@ func main() {
 			Name:  "workdir,w",
 			Usage: "path used to store large temporary data",
 		},
+		cli.StringFlag{
+			Name:  "runtime-root",
+			Usage: "root directory for the runtime",
+			Value: shim.RuncRoot,
+		},
+		cli.StringFlag{
+			Name:  "criu,c",
+			Usage: "path to criu",
```

Comment:
I guess it's ok to not have a default as, if I followed the code through, an empty/unset value will just not set the flag on `runc` leaving it up to runc to use a default in the case of checkpoint/restore actions. Is that correct? Any value in having the default here versus there? I'm thinking it doesn't matter as we weren't setting the `criu` binary path before.

Reply:
`go-runc` ignores the flag if it's empty meaning that `runc` will use its default. But you gave me a doubt for a few seconds :)

So, yes, I'd rather defer to the default built in `runc`